scripts/scripts.js

```diff
@@ -453,12 +453,12 @@ export function decorateExternalLinks(main) {
  */
 // eslint-disable-next-line import/prefer-default-export
 export function decorateMain(main) {
+  buildAutoBlocks(main);
+  decorateSections(main);
```

Comment:
`decoarateButtons` now depends on metadata added by `decorateSections`. I may have to come up with a different solution for this.